### PR TITLE
Fix artifact Download issue in release build

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

Fix artifact Download issue in release build

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/releaseBuild/azureDevOps/templates/nuget.yml [hunk 1]
```diff
@@ -39,11 +39,8 @@ jobs:
     displayName: 'Download PowerShell build artifacts'
     inputs:
       buildType: current
-      downloadType: specific
-      itemPattern: |
-        finalResults/PowerShell-*-win-*.zip
-        finalResults/powershell-*.tar.gz
-        results/powershell-*-osx-x64.tar.gz
+      downloadType: single
+      artifactName: finalResults
```
Comment (daxian-dbw) [inline tools/releaseBuild/azureDevOps/templates/nuget.yml]: Is `finalResults` a folder or a single file? Is this change actually copies all items in the `finalResults` folder to `$(System.ArtifactsDirectory)`?

**Review by daxian-dbw — APPROVED** [review]
Sign off to unblock the build. But I don't really understand the change 😃 Left a question below for my education.

File: tools/releaseBuild/azureDevOps/templates/nuget.yml (hunk 1 above)
Comment (TravisEz13) [reply tools/releaseBuild/azureDevOps/templates/nuget.yml]: This change, yes downloads all files from finalResults.  I always hit some issue if I tried the filters, and this was faster than what we had before so I stopped here.

Comment (TravisEz13) [reply tools/releaseBuild/azureDevOps/templates/nuget.yml]: You questions seems like you understood the change well enough.

Comment (daxian-dbw) [reply tools/releaseBuild/azureDevOps/templates/nuget.yml]: I saw `finalResults` appears in the build log as a folder :) A lucky guess

**Pull request merged**